config/content_misc.php

```diff
@@ -0,0 +1,56 @@
+<?php
```

Comment:
Yes, could not create comment without it right now, gives an error "Division by zero".